app-misc/remind/remind-05.03.02.ebuild

```diff
@@ -1,15 +1,15 @@
-# Copyright 1999-2024 Gentoo Authors
+# Copyright 1999-2025 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
 DESCRIPTION="Ridiculously functional reminder program"
 HOMEPAGE="https://dianne.skoll.ca/projects/remind/"
-SRC_URI="https://salsa.debian.org/dskoll/remind/-/archive/${PV}/${P}.tar.bz2"
+SRC_URI="https://dianne.skoll.ca/projects/remind/download/${P}.tar.gz"
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="amd64 ~ppc x86"
+KEYWORDS="~amd64 ~ppc ~x86"
 IUSE="tk"
 
 RDEPEND="
```

Comment:
The deps look wrong to me. Cairo and Pango are surely needed (imported) at runtime (as well as in `DEPEND` if configure checks for them). Please fix that in a followup.